src/redux/features/votes/votesSelectors.ts

```diff
@@ -0,0 +1,5 @@
+import { RootState } from 'src/redux/rootReducer';
+
+export const selectProjectVotes = (state: RootState) => state.votes.votes;
+
+export const selectProjectVotesRequest = (state: RootState, requestId: string) => state.votesRequests[requestId];
```

Comment:
selectProjectVotesEditRequest?

Reply:
So I added an async thunk to get the project votes that also uses the `votesRequests` store, seems like the naming should stay generic in this case?

Reply:
We should have separate slices for the edit requests and the get requests. Added a comment in the reducers section.

Example:
```
export const selectProjectVotes = (state: RootState, projectId: number) => state.votes[projectId];

export const selectProjectVotesEditRequest = (state: RootState, requestId: string) => state.votesRequests[requestId];
```

and the reducers something like this

```
// Define a type for the slice state
type Data = Record<number, StatusT<GetProjectVotesResponsePayload>>;

// Define the initial state
const initialState: Data = {};

export const votesSlice = createSlice({
  name: 'votesSlice',
  initialState,
  reducers: {},
  extraReducers: (builder: ActionReducerMapBuilder<Data>) => {
    buildReducers(requestProjectVotesGet, true)(builder);
  },
});

export const votesReducer = votesSlice.reducer;

type VotingRequestsState = Record<string, StatusT<boolean>>;

const initialVotingRequestsState: VotingRequestsState = {};

export const votesRequestsSlice = createSlice({
  name: 'votesRequestsSlice',
  initialState: initialVotingRequestsState,
  reducers: {},
  extraReducers: (builder: ActionReducerMapBuilder<VotingRequestsState>) => {
    buildReducers(requestProjectVotesUpdate)(builder);
    buildReducers(requestProjectVotesDelete)(builder);
  },
});

```

Have the async thunks return boolean values for the updates/deletes.